packages/perseus/src/components/graphie.tsx

```diff
@@ -187,6 +189,7 @@ class Graphie extends React.Component<Props> {
             });
         }
 
+        // @ts-expect-error - TS2339: Property 'snap' does not exist on type 'Graphie'.
         graphie.snap = this.props.options.snapStep || [1, 1];
```

Comment:
**thought**: It'd be nice if we passed this into `graphInit()` or `init()` instead of this direct assignment.  No change required, just musing on future improvements.